app.rb

```diff
@@ -0,0 +1,138 @@
+require './person'
+require './capitalize_decorator'
+require './trimmer_decorator'
+require './student'
+require './classroom'
+require './teacher'
+require './rental'
+require './book'
+
+class App
+  def initialize
+    @people = []
+    @books = []
+    @rentals = []
+  end
+
+  def list_all_books
+    puts 'There are no books to show! Please add a book.' if @books.empty?
+
+    @books.each { |book| puts "Title: \"#{book.title}\", Author: #{book.author}" }
+    puts
+    puts
+  end
+
+  def list_all_people
+    puts 'There are no people to show! Please add a student or a teacher.' if @people.empty?
+
+    @people.each { |person| puts "[#{person.class}] Name: #{person.name}, Id: #{person.id}, Age: #{person.age}" }
+    puts
+    puts
+  end
+
+  def create_a_person
+    print 'Do you want to create a student (1) or teacher (2)? [Input the number]: '
+    choice = gets.chomp
+
+    case choice
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'Invalid input! Please enter 1 or 2'
+    end
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+
+    print 'Name: '
+    name = gets.chomp
+
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp.downcase
+
+    parent_permission = permission == 'y'
+
+    student = Student.new('Grade 12', age, name, parent_permission)
+    @people << student
+
+    puts 'Person/Student created successfully'
```

Comment:
- [ ] It would be better if we are able to create a student data, in trying to create one I encountered an error I would like you to address. After entering the parent_permission, the app crashed as we can see in the screenshot below
![Screenshot from 2023-03-10 12-47-55](https://user-images.githubusercontent.com/46388113/224319260-2e5720ff-52c2-40d5-a8f3-0bd463b7dd17.png)